requirements.txt

```diff
@@ -1,8 +1,10 @@
 # Core
 pandas
-humanize
 pyyaml
 openpyxl
+odfpy
+simplejson
+humanize
```

Comment:
odfpy allows pandas to read Open Document format files the same way as it would excel files. It's an easy win.